test/shopify_liquid_test.rb

```diff
@@ -12,10 +12,10 @@ def test_deprecated_filter_alternatives
   end
 
   def test_filter_labels
-    assert_equal(171, ThemeCheck::ShopifyLiquid::Filter.labels.size)
+    assert_equal(137, ThemeCheck::ShopifyLiquid::Filter.labels.size)
   end
 
   def test_object_labels
-    assert_equal(83, ThemeCheck::ShopifyLiquid::Object.labels.size)
```

Comment:
Here are also some mismatches:

```ruby
YAML.load(File.read("./data/shopify_liquid/objects.yml")) - SourceIndex.objects.map(&:name)
["customer_address", "product_variant", "tablerow"]

SourceIndex.objects.map(&:name) - YAML.load(File.read("./data/shopify_liquid/objects.yml"))
[
  "media",
  "app",
  "discount",
  "color",
  "company_address",
  "company",
  "company_location",
  "country",
  "filter",
  "filter_value",
  "focal_point",
  "generic_file",
  "image_presentation",
  "tablerowloop",
  "measurement",
  "money",
  "selling_plan_price_adjustment",
  "rating",
  "robots",
  "group",
  "rule",
  "selling_plan_allocation_price_adjustment",
  "selling_plan_checkout_charge",
  "selling_plan_group_option",
  "selling_plan_option",
  "user",
  "form_errors",
  "pending_payment_instruction_input",
  "predictive_search_resources",
  "sitemap",
  "sort_option",
  "transaction_payment_details",
  "user_agent",
  "array",
  "string",
  "number",
  "boolean",
  "nil",
  "empty"
]
```

Reply:
I like that approach, so I added `customer_address` and `product_variant`.
`tablerow` will not be added, as I noticed that it is a _tag_, and is therefore already being suggested.